ansible/ansible-ai-connect-service
# AAP-21297: Reword the telemetry settings page in admin portal

[Jira Issue: <https://issues.redhat.com/browse/AAP-21297>](https://issues.redhat.com/browse/AAP-21297)


## Description
Reword the telemetry settings page in admin portal

## Testing
### Steps to test
1. Pull down the PR
2. `cd ansible_wisdom_console_react/`
3. `npm install && npm start`
4. Check the admin portal rewording as:

![image-2024-03-05-13-19-09-338](https://github.com/ansible/ansible-wisdom-service/assets/4602417/11c89b4e-619e-4188-b6c8-910c4757938a)

### Scenarios tested
Above steps

## Production deployment
